app/assets/javascripts/darkswarm/services/products.js.coffee

```diff
@@ -24,6 +24,10 @@ Darkswarm.factory 'Products', ($resource, Enterprises, Dereferencer, Taxons, Pro
         if product.variants?.length > 0
           prices = (v.price for v in product.variants)
           product.price = Math.min.apply(null, prices)
+          if product.variants.length > 1
```

Comment:
alternatively
```
product.variant_names = product.variants?.map(function (v) { return v.name_to_display }).join(' ');
```
which can even be done in `filter_products.js.coffee` to avoid touching more than one file